core/lib/via_btc_client/src/indexer/parser.rs

```diff
@@ -452,6 +507,31 @@ fn find_via_inscription_protocol(instructions: &[Instruction]) -> Option<usize>
     position
 }
 
+pub fn get_btc_address(common_fields: &CommonFields, network: Network) -> Option<Address> {
+    let internal_pubkey =
+        UntweakedPublicKey::from_slice(&common_fields.encoded_public_key.as_bytes()).ok()?;
```

Comment:
`&common_fields.encoded_public_key.as_bytes()`

.as_bytes() itself return reference so we can remove & from this input.